### PR TITLE
Add record trimmer

### LINKED ISSUE #6: Trim too big record

### PR DESCRIPTION
**Pull request by RayBB**
Fixes #6

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		"description": "meta[name=\"description\"]",
 		"text": "h1, h2, h3, h4, h5, h6, p, li"
 	},
+	"trimSize": 10000,
```
Comment (nitriques) [inline config.json]: I think we should name this `maxRecordSize`. Also, can you add a description of the option in the read me please ?
Thanks a lot for your contribution !

**Comment by RayBB** [comment]
Added changes

**Comment by nitriques** [comment]
Thanks @RayBB

**Comment by nitriques** [comment]
It's on npm as `2.1.0` !

**Pull request merged**